docker/dm_keys/dm

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I think this would be better to add a .gitignore on this folder in order to not put the keys in SCM and then put a note on README on how to generate the keys or add key generation as part of the deploy script

Reply:
Regarding "variable substitution" option for the images, I think the best place to change them if you are working on Red PNDA is to directly edit the docker-compose.yml and customize it for you.
Or if you want to explicitly have a place to put only image versions, I can create a docker-compose [.env file](https://docs.docker.com/compose/environment-variables/#the-env-file).